### PR TITLE
Fix flickering in Gauntlet Legends

### PR DESCRIPTION
**Pull request by loganmc10**
Fixes https://github.com/gonetz/GLideN64/issues/1705

@fzurita could you test and confirm?

### CODE REVIEW

**Comment by fzurita** [comment]
Sure, once I get a chance.

**Comment by fzurita** [comment]
I can confirm that this fixes the flickering issue.

File: src/RSP.cpp [hunk 1]
```diff
@@ -303,7 +303,7 @@ void RSP_Init()
 		config.generalEmulation.hacks |= hack_texrect_shade_alpha;
 	else if (strstr(RSP.romname, (const char *)"THE LEGEND OF ZELDA") != nullptr ||
 			 strstr(RSP.romname, (const char *)"ZELDA MASTER QUEST") != nullptr)
-		 config.generalEmulation.hacks |= hack_subscreen | hack_ZeldaMonochrome;
+		config.generalEmulation.hacks |= hack_subscreen | hack_ZeldaMonochrome;
```
Comment (gonetz) [inline src/RSP.cpp]: What is changed in this line?

Comment (fzurita) [reply src/RSP.cpp]: Looks like a formatting fix.

Comment (loganmc10) [reply src/RSP.cpp]: Yeah it was just an extra space I spotted

Comment (gonetz) [reply src/RSP.cpp]: Ok

**Comment by gonetz** [comment]
Thanks for the fix!

**Pull request merged**